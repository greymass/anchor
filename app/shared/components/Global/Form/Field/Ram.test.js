import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GlobalFormFieldRam from './Ram';

const mockProps = {
  icon: 'icon',
  label: 'multi token field',
  name: 'multi token field',
  readOnly: false,
};

describe('GlobalFormFieldRam', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GlobalFormFieldRam {...mockProps} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
