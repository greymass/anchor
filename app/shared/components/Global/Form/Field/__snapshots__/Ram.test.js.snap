// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalFormFieldRam renders correctly 1`] = `
<FormField
  control={[Function]}
  icon="icon"
  label="multi token field"
  name="multi token field"
  onChange={[Function]}
  placeholder="0"
  readOnly={false}
/>
`;
